docs/adr/0006-monitoring-structure.md

```diff
@@ -0,0 +1,69 @@
+# Application Monitoring in Operate First using Prometheus
+
+* Status: accepted
+* Deciders: 4n4nd, anishasthana, hemajv, HumairAK, tumido
+* Date: 2021-01-14
+
+Technical Story: [issue-1](https://github.com/operate-first/blueprint/issues/8)
+                 [issue-2](https://github.com/operate-first/apps/issues/57)
+
+## Context and Problem Statement
+
+Since there will be multiple services deployed in the Operate First environment
+(ex. Jupyterhub, Argo, Superset, Observatorium, etc) distributed over various
+namespaces, we need to be able to monitor them.
+
+To do so we will use [Prometheus](https://prometheus.io/).
+This will be deployed using the [ODH Operator](https://opendatahub.io/) (ODH
+operator includes the Prometheus Operator to do this).
+
+Normally, Prometheus Operator would have been able to monitor services across
+namespaces using cluster-wide access, but the recent versions of Openshift have
+an instance of Prometheus Operator pre-installed for cluster monitoring so we
```

Comment:
> but the recent versions of Openshift have
an instance of Prometheus Operator

change to: 

> but the recent versions of Openshift **has** an instance of Prometheus Operator

Reply:
> Normally, Prometheus Operator would have been able to monitor services across namespaces using cluster-wide access, but the recent versions of Openshift have an instance of Prometheus Operator pre-installed for cluster monitoring so we cannot do a cluster-wide Prometheus Operator setup since this would conflict with the cluster monitoring instance.

This is sentence is a bit long, can we break it up, suggestion: 

> Normally, Prometheus Operator would have been able to monitor services across namespaces using cluster-wide access, but the recent versions of Openshift have an instance of Prometheus Operator pre-installed for cluster monitoring. As such, we cannot do a cluster-wide Prometheus Operator setup since this would conflict with the cluster monitoring instance.